README.md

```diff
@@ -214,6 +220,103 @@ The [SolaceProtoBuffRecordTest](solace-apache-beam-samples/src/main/java/com/sol
     ```
 1. Verify that the messages are being outputted in the log.
 
+#### SolaceBigQuery
+
+The [SolaceBigQuery](solace-apache-beam-samples/src/main/java/com/solace/connector/beam/examples/SolaceBigQuery.java) example receives a message from PubSub+ queue, transforms it, and then writes it to a BigQuery table. 
+
+1. Create a BigQuery table with appropriate schema:
+```
+[
+ {
+ 
+   "name": "date",
+   "type": "STRING",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "sym",
+   "type": "STRING",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "time",
+   "type": "STRING",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "lowAskSize",
+   "type": "INTEGER",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "highAskSize",
+   "type": "INTEGER",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "lowBidPrice",
+   "type": "FLOAT",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "highBidPrice",
+   "type": "FLOAT",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "lowBidSize",
+   "type": "INTEGER",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "highBidSize",
+   "type": "INTEGER",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "lowTradePrice",
+   "type": "FLOAT",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "highTradePrice",
+   "type": "FLOAT",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "lowTradeSize",
+   "type": "INTEGER",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "highTradeSize",
+   "type": "INTEGER",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "lowAskPrice",
+   "type": "FLOAT",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "highAskPrice",
+   "type": "FLOAT",
+   "mode": "NULLABLE"
+ },
+ {
+   "name": "vwap",
+   "type": "FLOAT",
+   "mode": "NULLABLE"
+ }
+]
+```
+1. Run the SolaceBigQuery sample on Google Dataflow:
+```shell script
+mvn compile exec:java -Dexec.mainClass=com.solace.connector.beam.examples.SolaceBeamBigQuery -Dexec.args="--sql=Q/fx-001 --cip=${SOLACE_URI} --cu=${SOLACE_USERNAME} --cp=${SOLACE_PASSWORD} --vpn=${SOLACE_VPN} --project=${GCP_PROJECT} --bigQueryProject=${BIGQUERY_PROJECT} --bigQueryDataset=${BIGQUERY_DATASET} --bigQueryTable=${BIGQUERY_TABLE} --tempLocation=${GOOGLE_STORAGE_TMP} --workerMachineType=n1-standard-2 --runner=DataflowRunner --autoscalingAlgorithm=THROUGHPUT_BASED --maxNumWorkers=4 --stagingLocation=${GOOGLE_STORAGE_STAGING}" -Pdataflow-runner
```

Comment:
`-Dexec.mainClass=com.solace.connector.beam.examples.SolaceBeamBigQuery` needs to be -`Dexec.mainClass=com.solace.connector.beam.examples.SolaceBigQuery`